packages/containers/src/Redirect/Redirect.test.js

```diff
@@ -1,20 +1,24 @@
+import React from 'react';
+import { mount } from 'enzyme';
 import Redirect from './Redirect.container';
 
 describe('Redirect', () => {
 	it('should dispatch a redirect action', () => {
 		const dispatch = jest.fn();
-		const result = new Redirect.WrappedComponent({ path: '/hello-world', dispatch });
+		const Wrapped = Redirect.WrappedComponent;
+		const wrapper = mount(<Wrapped path="/hello-world" dispatch={dispatch} />);
 		expect(dispatch.mock.calls.length).toBe(1);
 		const action = dispatch.mock.calls[0][0];
 		expect(action.type).toBe('REDIRECT');
 		expect(action.cmf.routerReplace).toBe('/hello-world');
-		expect(result).toMatchSnapshot();
+		expect(wrapper.childAt(0).getElement()).toMatchSnapshot();
```

Comment:
is it possible to take this opportunity to replace this snapshot by a specific check?